FprimeZephyrReference/Components/ImuManager/ImuManager.fpp

```diff
@@ -2,7 +2,6 @@ module Components {
     port AccelerationGet(ref condition: Fw.Success) -> Drv.Acceleration
     port AngularVelocityGet(ref condition: Fw.Success)-> Drv.AngularVelocity
     port MagneticFieldGet(ref condition: Fw.Success) -> Drv.MagneticField
-    port TemperatureGet(ref condition: Fw.Success) -> F64
```

Comment:
This functionality was removed in the transition from discrete sensor components to the ImuManager. Removing here as well and removing the test.